pom.xml

```diff
@@ -56,6 +56,7 @@
         <java.level>8</java.level>
         <groovy.version>2.4.7</groovy.version>
         <jenkins.version>2.60.3</jenkins.version>
+        <spotbugs.failOnError>false</spotbugs.failOnError>
```

Comment:
If we're now using spotbugs, we shouldn't retain the config for findbugs.
We should be using one or the other; we shouldn't be configuring both.

Reply:
I had a lot of spotbugs errors after upgrading the parent pom. Fixing these errors has nothing to do with this PR. And I don't have the time to do it. That's why I just configured it to not fail the build.

I had to upgrade the parent pom because some dependencies are now using Multi-Release JARs, which was not supported with the older version of the parent pom.

Reply:
I agree that fixing all the spotbugs errors isn't part of this PR; that's not what I'm asking for here.

What I was complaining about was that, after switching _from_ findbugs _to_ spotbugs, we need to _remove_ the old "tell findbugs not to fail on error" config and replace it with a "tell spotbugs not to fail on error" config.
At present, we've got two "tell ... not to fail on error" configs in the pom.xml - if we're not using findbugs anymore then we can (and should) remove the findbugs-specific configuration.
(but we should probably retain the TODO comment, as fixing all of findbugs/spotbugs' complaints is something that ought to be done at some point, just not as part of this PR)

i.e. remove the findbugs config (line 226 in the original pom) and add a `<!-- FIXME need to review spotbugs errors -->` comment to the spotbugs config.